### PR TITLE
Setup: Project vs hours endpoint

### PR DESCRIPTION
**Pull request by iCHAIT**

### CODE REVIEW

File: views/layout.erb [hunk 1]
```diff
@@ -16,14 +16,13 @@
...
+    <link href="dist/css/styles.css" rel="stylesheet">
 
     <!-- Custom Fonts -->
     <link href="bower_components/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
 
     <script src="bower_components/d3/d3.min.js" charset="utf-8"></script>
-    <script src="bower_components/nvd3/build/nv.d3.js"></script>
-    <script src="bower_components/nvd3/build/nv.d3.css"></script>
-    <script src="bower_components/stream_layers/index.js"></script>
+    <script src="https://d3js.org/d3.v3.min.js"></script>
```
Comment (Kennric) [inline views/layout.erb]: is this the same as line 24?

Comment (iCHAIT) [inline views/layout.erb]: Sorry, but didn't get your point.

Comment (iCHAIT) [inline views/layout.erb]: Yeah, removing.

Comment (iCHAIT) [inline views/layout.erb]: @Kennric Fixed

**Comment by Kennric** [comment]
+1

**Pull request merged**